packages/flutter_tools/tool/daemon_client.dart

```diff
@@ -67,11 +67,13 @@ Future<void> main() async {
       _send(<String, dynamic>{'method': 'device.getDevices'});
     } else if (line == 'emulators') {
       _send(<String, dynamic>{'method': 'emulator.getEmulators'});
-    } else if (words.first == 'emulator-launch') {
+    } else if (words.first == 'launch') {
```

Comment:
is `launch` the same as  `emulator-launch`?  

@jonahwilliams can this break something?

Reply:
I changed that because its either here wrong or line 19. I can of course change it back if wanted

Reply:
FYI @DanTup , not sure if this is used

Reply:
I think line 19 (the comment) is probably wrong - it's probably better that this is more explicit (and changing the comment doesn't change behaviour). My understanding is that this is a simple CLI tool to manually interact with the daemon (for debugging/testing purposes).